docs/components/checkboxes.md

```diff
@@ -0,0 +1,226 @@
+<!--docs:
+title: "Material selection controls: CheckBoxes"
+layout: detail
+section: components
+excerpt: "Selection controls allow the user to select options."
+iconId:
+path: /catalog/SelectionControlsCheckBoxes/
+-->
+
+# Selection controls: checkboxes
+
+[Selection controls](https://material.io/components/selection-controls#usage) allow the user to select options.
+
+Use checkboxes to:
+
+- Select one or more options from a list
+- Present a list containing sub-selections
+- Turn an item on or off in a desktop environment
+
+![Checkbox hero example for menu options](assets/checkboxes/checkbox-hero.png)
+
+## Contents
+
+- [Contents](#contents)
+- [Using checkboxes](#using-checkboxes)
+- [Checkboxes](#checkboxes)
+- [Checkbox key properties](#checkbox-key-properties)
+- [Theming](#theming)
+
+## Using checkboxes
+
+Checkboxes allow the user to select one or more items from a set. Checkboxes can be used to turn an option on or off.
+
+### Making checkboxes accessible
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+## Checkboxes
+
+### Checkboxes example
+
+`CheckBox`
+- [Class description](https://api.flutter.dev/flutter/material/Checkbox-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/checkbox.dart)
+- [Demo site](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/checkboxes/regular)
+
+The following example should five checkboxes. The first one is enabled, the second one has tristate enabled, and the last one is disabled.
+
+![Checkboxes example.](assets/checkboxes/checkboxes-example.png)
+
+```dart
+Column(
+  children: [
+    for (var i = 0; i < 5; i += 1)
+      Row(
+        children: [
+          Checkbox(
+            onChanged: i == 4 ? null : (bool value) {
+              setState(() {
+                checked[i] = value;
+              });
+            },
+            tristate: i == 1,
+            value: checked[i],
+            activeColor: Color(0xFF6200EE),
+          ),
+          Text('Checkbox ${i + 1}')
```

Comment:
In my radio button docs, I also change the text color for the disabled row, I think that may look a little better:

https://github.com/material-components/material-components-flutter/blob/master/docs/components/radio_buttons.md

I suggest doing it for all the examples and the dartpad code

Reply:
That makes sense. Updating now.